src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java

```diff
@@ -1142,6 +1142,63 @@ public void testWrongtype() throws SQLException {
+        beforeEachSetup();
+
+        Geometry geomWKT = Geometry.STGeomFromText(wktLargeGeometry, 0);
```

Comment:
I don't it matters much, but these coordinates are in SRID 28992

Reply:
Thanks. For the purpose of this test, it shouldn't make a difference.